Detectors/TRD/base/include/TRDBase/PadPlane.h

```diff
@@ -71,9 +74,25 @@ class PadPlane
 
   GPUd() int getPadRowNumber(double z) const;
   GPUd() int getPadRowNumberROC(double z) const;
+  GPUd() double getPadRow(double z) const;
```

Comment:
Since there is also the function to get the pad row number as an `int` above, should we maybe rename this method to something like `getPadRowAsFloat`? And similar for `getPad`

Reply:
Hmm, I thought the word `Number` in the other functions made it clear that they return integers. Then, by excluding that word in the other functions, we make it clear that they are not integers. No problem if you disagree, I can change it in that case.

Reply:
Maybe it is just me. There is also a `getPadRow` method in the [Tracklet64 class](https://github.com/AliceO2Group/AliceO2/blob/b3920fd266e4d202a116ed2053cc97ff1cd003c8/DataFormats/Detectors/TRD/include/DataFormatsTRD/Tracklet64.h#L85) returning an integer maybe that confused me. I don't have a really strong opinion. So you can name it as you prefer.